_includes/tutorials/connect-sink-timestamp/ksql/markup/dev/select-from-stream-unix.adoc

```diff
@@ -0,0 +1,11 @@
+To convert the `BIGINT` `UTC` time into a `TIMESTAMP` `American/Denver` time, we need to use the `CONVERT_TZ` ksqlDB function. This function takes three arguments: a `TIMESTAMP` column, a string of the original timezone, and a string of the target conversion timezone. As it stands right now, we do no have a column that is a `TIMESTAMP` data type. Prior to using the `CONVERT_TZ` ksqlDB function, we need to convert the `EVENTTIME` column from a `BIGINT` to a `TIMESTAMP`. We can do that with the `FROM_UNIX` ksqlDB function. Run the command below to see what the `EVENTTIME` column will look like when converted to a `TIMESTAMP`.
```

Comment:
Minor point - I found it a little confusing that `CONVERT_TZ` is described and referenced a couple of times, but then the text goes on to use `FROM_UNIX 

Maybe say something along the lines of "Our first step is to convert the `BIGINT` into a `TIMESTAMP` " and move everything about `CONVERT_TZ` to the next section where the tutorial uses the `CONVERT_TZ` function.

But this could stay as is, I'll leave it to you.

Reply:
I can see how that is confusing. I added a comment to address this in my latest commit. https://github.com/confluentinc/kafka-tutorials/pull/760/commits/9904d2ac9ffb1d0894c431caf88c786ba19ad091